static/controllers/favoritesController.js

```diff
@@ -1,94 +1,94 @@
-app.controller('favoritesController', function ($scope, $localStorage, $http, FavoritesService, UtilityService) {
-    'use strict';
+app.controller('favoritesController', function ($scope, $localStorage, $http, FavoritesService, UtilityService, $interval, PredictionService) {
+    var vm = this;
```

Comment:
Actually necessary surprisingly! To keep the scope consistent (https://johnpapa.net/angularjss-controller-as-and-the-vm-variable/)